tests/integration/targets/vmware_ops_upload_iso_to_datastore_test/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+run_on_simulator: false
+
+upload_iso_to_datastore_datacenter: "{{ vcenter_datacenter }}"
+upload_iso_to_datastore_datastore: "{{ shared_storage_01 }}"
+upload_iso_to_datastore_src:  "/tmp/{{ tiny_prefix }}-test.iso"
+upload_iso_to_datastore_dst: "/{{ tiny_prefix }}-upload_iso_to_datastore/test.iso"
```

Comment:
I think it's better to save the `folder` in a separate variable and use it here and in `Delete a VM folder on given datacenter` task